typescript-react/src/components/Form/Form.module.css

```diff
@@ -21,12 +21,20 @@
     padding: 10px;
     box-shadow: -1px -1px 3px 0 btnPrimaryColor;
     background-color: loginBodyColor;
-    z-index: 5;
+    z-index: 110;
   }
 
   .FormComponent {
+    margin-bottom: 72px;
     padding-bottom: 64px;
   }
+
+  :global .MobileBody {
```

Comment:
I'm a bit rusty, but can't you instead (in your local app) do `:global .MobileBody .dslForm`?

Reply:
I'll double check. Thanks

Reply:
Removed irrelevant code from PR. Thanks.